tests/components/advantage_air/test_climate.py

```diff
@@ -46,6 +56,7 @@ async def test_climate_async_setup_entry(hass, aioclient_mock):
     assert state.state == HVAC_MODE_FAN_ONLY
     assert state.attributes.get("min_temp") == 16
     assert state.attributes.get("max_temp") == 32
+    assert state.attributes.get("temperature") == 24
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)